turbopack/crates/turbopack-core/src/issue/mod.rs

```diff
@@ -452,15 +453,19 @@ impl IssueSource {
     }
 
     #[turbo_tasks::function]
-    pub fn from_line_col(
+    pub async fn from_line_col(
         source: Vc<Box<dyn Source>>,
         start: SourcePos,
         end: SourcePos,
-    ) -> Vc<Self> {
-        Self::cell(IssueSource {
+    ) -> Result<Vc<Self>> {
+        // If we have a source map, map the line/column to the original source.
+        let start = source_pos(source, start.line, start.column).await?;
+        let end = source_pos(source, end.line, end.column).await?;
+
+        Ok(Self::cell(IssueSource {
             source,
```

Comment:
If it's a `Token::Original` you also need to lookup the original source from the source map and use this instead. So the line/column points to the original source code